src/core/projectinfo.cpp

```diff
@@ -743,3 +744,39 @@ QVariantMap ProjectInfo::getCopyrightDecorationConfiguration()
 
   return configuration;
 }
+
+QVariantMap ProjectInfo::getImageDecorationConfiguration()
+{
+  QVariantMap configuration;
+  const QString configurationName = QStringLiteral( "Image" );
```

Comment:
All those literals are defined the same way in QGIS as well? I think ideally they should be some const or are they by purpose not made part of the interface in QGIS?

Reply:
@domi4484 , correct, they are.

That's in big part because decorations have been living in src/app for as long as they've existed.